bin/import.ml

```diff
@@ -127,13 +127,17 @@ module Scheduler = struct
 
   let go ~(common : Common.t) ~config:dune_config f =
     let stats = Common.stats common in
-    let config = Dune_config.for_scheduler dune_config stats in
+    let config =
```

Comment:
I'm confused: why are you explicitly choosing `React` here and in a few other places? Shouldn't you take it from the Dune configuration instead? (Also, I don't see where the latter defauls to `Ignore`.)

Reply:
Just noticed as I was writing the test. I've fixed this thanks.

There's a few other places where the value is arbitrarily chosen because we're not running watch mode (e.g. when running `dune subst`).

> Shouldn't you take it from the Dune configuration instead?

You mean Dune_config? That corresponds to the `dune-config` and I don't see why we need to add this option there.

Reply:
Looks good now, thanks!